### ISSUE TITLE
Gloves Texture seems to be hardcoded

### ISSUE THREAD

**Issue by JuceDoesThings** [emilyploszaj/trinkets#127]
I recently downloaded a mod (Things) That uses your gloves trinket. However, when I tried to change the texture as it is in Trinkets (assets/trinkets/textures/gui/slots/glove AND assets/trinkets/textures/gui/slots/glove_mirrored) the textures did not change at all. This isn't a massive issue, but it looks ugly without me being able to modify it. For a visual, [this is how it looks](https://cdn.discordapp.com/attachments/649513555078545408/908845385299140638/unknown.png)

**Comment by emilyploszaj** [comment]
all textures can be modified for almost every mod. you simply need to make a resource pack with a new texture at the proper location. it sounds like you tried to modify the mod? which might work, but is bad for a lot of other reasons. resource packs are quite simple, and you already know the path.

**Comment by JuceDoesThings** [comment]
I actually DID use a resource pack! It just didn't work, even though I used a resource pack. Here, I'll send you a [resource pack](C:\Users\Raymond\AppData\Roaming\.minecraft\resourcepacks) with just the broken texture. Sorry if my wording was confusing. if you would like me do be more specific, I inserted this as a part of my personal resource pack in hopes of making Trinkets work with my already large resource pack and for some reason the glove texture SPECIFICALLY didn't work. Also, this isn't an issue with just this mod. I've had similar issues with BetterEnd. I mean maybe this is some mod incompatibility but to me that wouldn't make sense. Especially as it works for every other Trinket slot
